noroute_mesh/neighb[] data